src/dmd/todt.d

```diff
@@ -76,9 +76,9 @@ extern (C++) void Initializer_toDt(Initializer init, ref DtBuilder dtb)
         dtb.nzeros(cast(uint)vi.type.size());
     }
 
-    void visitStruct(StructInitializer si)
+    void visitStruct(StructInitializer /* unused */)
```

Comment:
I would not rename the variables, commenting them is enough indication of them being unused.

Reply:
Done